main/res/values/preference_keys.xml

```diff
@@ -253,6 +253,9 @@
     <string translatable="false" name="pref_su_tokenpublic">su_tokenpublic</string>
     <string translatable="false" name="pref_temp_su_token_secret">su-temp-token-secret</string>
     <string translatable="false" name="pref_temp_su_token_public">su-temp-token-public</string>
+    <string translatable="false" name="pref_cache_filter_config">cache_filter_config</string>
```

Comment:
the `cache_filter_config` must be the result of an wrong resolution of a merge conflict. At least I have not intentionally added that line.

Reply:
@baiti I recently added this preference key to master for the new cache filter. It should not be on release, but it doesn't hurt there either. 

Maybe this was somehow left over innyour local copy when you switched between master and release?